lib/Runtime/Provisioner/Provisioner.cpp

```diff
@@ -73,6 +75,16 @@ llvm::Error Provisioner::provision(DAGListTy &networks, Module &module) {
       }
     }
   }
+  if (cctx.backendOpts.collectConstants) {
+    DEBUG_GLOW(llvm::dbgs()
```

Comment:
Could also use glog (e.g. `LOG(INFO)`) since we're moving toward it everywhere.

Reply:
ah right yeah, that helps a lot. What i really wanted was LOG_EVERY_MS or LOG_EVERY_N to not flood the log in high traffic scenarios.